search/populate.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from io import BytesIO
+from typing import Union
+
+from flask import Blueprint, Response
+
+import requests
+from elasticsearch import Elasticsearch
+from lxml import etree
+
+from search.constants import (CONTENT_SERVICES_TO_INDEX, ELASTICSEARCH_HOSTS,
+                              GATEWAY_URL)
+from search.utils import strip_slashes
+
+LOGGER = logging.getLogger(__name__)
+
+
+def request_data(url: str) -> Union[requests.Response, None]:
+    try:
+        response = requests.get(url)
+        response.raise_for_status()
+        return response
+    except requests.HTTPError as error:
+        LOGGER.exception('%s %s: Unable to retrieve content from %s.',
+                         error.response.reason,
+                         error.response.status_code,
+                         url)
+    return None
+
+
+def index_data(index: str, doc_type: str, id: str, body: dict) -> None:
+    try:
+        search = Elasticsearch(hosts=ELASTICSEARCH_HOSTS)
+        search.index(index=index, doc_type=doc_type, id=id, body=body)
+        LOGGER.debug('Document %s saved at /%s/%s/%s', id, index, doc_type, id)
+    except ConnectionError as error:
+        LOGGER.exception('Unable to index /%s/%s/%s: %s', index, doc_type, id, error)
+
+
+def get_populate_blueprint() -> Blueprint:
+    blueprint = Blueprint('populate', __name__)
+
+    @blueprint.route('/populate', methods=['POST'])
+    def populate() -> Response:
+        """
+        Route for triggering indexing of known paths. Will be used as a proof of
+        concept. This should be removed before MVP release.
+        """
+        gateway = strip_slashes(GATEWAY_URL)
+        document_count = 0
+
+        for path in CONTENT_SERVICES_TO_INDEX:
```

Comment:
`name` or `prefix`, as it's not really a `path` (ends up forming part of it).

Reply:
how about `service`?